lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -56,7 +55,7 @@ class DOM {
    * @throws {Error}
    */
   cloneTemplate(selector, context) {
-    const template = context.querySelector(selector);
+    const template = /** @type {HTMLTemplateElement} */ (context.querySelector(selector));
```

Comment:
compiling this PR in devtools i get this error:

```
/Users/paulirish/chromium/src/third_party/WebKit/Source/devtools/front_end/audits2/lighthouse/renderer/dom.js:58: ERROR - Type "HTMLTemplateElement" nullability not marked explicitly with "?" (nullable) or "!" (non-nullable)
    const template = /** @type {HTMLTemplateElement} */ (context.querySelector(selector));
```

that's the only error though!